task/07-yield-tasks.js

```diff
@@ -145,7 +200,38 @@ function* mergeSortedSequences(source1, source2) {
  *   Most popular implementation of the logic in npm https://www.npmjs.com/package/co
  */
 function async(generator) {
-    throw new Error('Not implemented');
+    
+    let iterator = generator();
+
```

Comment:
Yeah.. it makes sense to add notes here:
The task is for Promise and Generators understanding so do not use async/await feature for the task.